src/global-firestore-data/courses.ts

```diff
@@ -81,21 +83,6 @@ const getCourse = async (
   }
   return course.data()?.course;
 };
-
-/**
- * This function transforms semester and year to a roster ID. EX: Spring 2023 -> SP23
- * */
-const seasonAndYearToRosterIdentifier = (season: FirestoreSemesterSeason, year: number): string => {
-  const seasonToSemesterMap = {
-    Fall: 'FA',
-    Spring: 'SP',
-    Winter: 'WI',
-    Summer: 'SU',
-  } as const;
-
-  return `${seasonToSemesterMap[season]}${year - 2000}`;
-};
```

Comment:
Why was this removed?

Reply:
I think this implementation is better than the existing one

Reply:
Gotcha, i updated the implementation, but still removed the duplicate one.